package.json

```diff
@@ -1,53 +1,46 @@
 {
   "name": "@xoi/gps-metadata-remover",
-  "version": "1.1.2",
+  "version": "2.0.0",
   "description": "Removes GPS metadata from images and videos",
   "repository": {
     "type": "git",
     "url": "ssh://git@github.com:xoeye/gps-metadata-remover.git"
   },
-  "main": "lib/index.js",
+  "main": "build/index.js",
   "author": "alindner@xoi.io",
   "license": "apache",
   "homepage": "https://github.com/xoeye/gps-metadata-remover",
   "bugs": {
     "url": "https://github.com/xoeye/gps-metadata-remover/issues"
   },
   "files": [
-    "lib"
+    "build"
   ],
   "dependencies": {
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
     "Base64": "^1.1.0",
-    "debug": "^4.1.1",
-    "jdataview": "^2.5.0"
+    "crc": "^4.3.2",
+    "debug": "^4.1.1"
   },
   "devDependencies": {
-    "@babel/cli": "7.18.10",
-    "@babel/core": "^7.11.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.11.0",
-    "@babel/preset-env": "^7.11.0",
-    "@babel/preset-flow": "^7.11.0",
-    "babel-plugin-transform-remove-console": "^6.9.4",
-    "flow-bin": "^0.109.0",
-    "flow-copy-source": "^2.0.9",
     "ffprobe": "^1.1.0",
-    "ffprobe-static": "^3.0.0",
+    "ffprobe-static": "^3.1.0",
     "gm": "^1.23.1",
     "hasha": "^5.2.0",
     "is-corrupted-jpeg": "^1.0.2",
-    "jest": "^26.6.3",
```

Comment:
this diff hits like a warm shower after a long day of working outside in the sun.